packages/wonder-blocks-button/components/button-core.js

```diff
@@ -129,7 +129,7 @@ const sharedStyles = StyleSheet.create({
         outline: "none",
         textDecoration: "none",
         boxSizing: "border-box",
-        "::-moz-focus-inner": {
+        "::MozFocusInner": {
```

Comment:
For reasons I haven't fully investigated yet, changing it to this doesn't apply the style correctly to the component. https://deploy-preview-262--wonder-blocks.netlify.com/#button-1 on Firefox still has the inner focus ring.